restore-cognito.md

```diff
@@ -0,0 +1,32 @@
+Prerequisites 
+For creating and executing backups, you can use the npm package called `cognito-backup-restore` 
+(do `npm install cognito-backup-restore` to get the package)
+
+Required IAM permissions
+If you want to create a backup of the users you will need the permission - `ListUsers`
+Restoring the users requires - `AdminCreateUser`
+There might be more roles required to do all the operations for getting and restoring users. If you need a permission added
+speak with Iva or Pol, who will add it through IAM.
+
+**Downloading user data**
+Here do either 1 or 2 depending on the situation.
+
+1. In case there is no cognito user pool or the user pool is empty, we have weekly user backup which executes every sunday at 1am. 
+To find the latest users backup file you need to go to biomage-backups-production S3 bucket and find the latest object in the bucket. 
```

Comment:
Instead of saying `to find the latest users ....` I would say `You will need to find the latesst users backup file/. To do it, you need to ...`. In that way, the runbook is more explicit that finding the latest users backup file is the required next step. In runbooks, you need to be clear, explicit and use short and easy to understand sentences. This is because the intended usage of runbooks is in situations when something went wrong, and hence people are under stress and they will be more likely to misread sentences or make mistakes. Well written runbook minimises the probability of such mistakes happening.